File: projects/code_combat/1_Kithgard_Dungeon/046-Cupboards_of_Kithgard_B/cupboardsOfKithgardB.js
// There may be something around to help you!
// First, move to the Cupboard.
for (var i = 0; i < 2; i++) {
    hero.moveRight();
    hero.moveDown();
}
hero.moveDown();

// Then, attack the "Cupboard" inside a while-true loop.
while(true) {
    hero.attack("Cupboard");
}

File: projects/code_combat/1_Kithgard_Dungeon/046-Cupboards_of_Kithgard_B/cupboards_of_kithgard_b.py
# You need the Elemental codex 1+ to cast "Haste"
# You need an unique hero to perform "Mana Blast"
# You need the Emperor's gloves to cast "Chain Lightning"

hero.cast("haste", hero)
hero.attack("Cupboard")
hero.moveRight(0.7)
hero.moveDown()
hero.moveRight(2.6)
hero.cast("chain-lightning", hero.findNearestEnemy())
hero.manaBlast()
